frontend/components/facebook.js

```diff
@@ -120,7 +131,7 @@ class Facebook {
                     timestamp: Date.now(),
                     sender:
                       {
-                        id: '1397905100304615',
+                        id: '2178896222126069',
```

Comment:
Hmm - I wonder if there would be any way to make it so it could pull this from config.json? or something that supports multiple IDs

Reply:
I actually fixed this in a different PR too i think - sync the latest master!